src/authn/ccloudProvider.test.ts

```diff
@@ -110,16 +111,17 @@ describe("authn/ccloudProvider.ts ConfluentCloudAuthProvider methods", () => {
   });
 
   it("createSession() should update the auth status secret on successful authentication", async () => {
-    const setSecretStub = sandbox.stub(getStorageManager(), "setSecret").resolves();
+    const setCCloudAuthStatusStub = sandbox
+      .stub(ResourceManager.getInstance(), "setCCloudAuthStatus")
+      .resolves();
     getCCloudConnectionStub.resolves(TEST_AUTHENTICATED_CCLOUD_CONNECTION);
     // authentication completes successfully
     browserAuthFlowStub.resolves({ success: true, resetPassword: false });
 
     await authProvider.createSession();
 
     sinon.assert.calledWith(
-      setSecretStub,
-      SecretStorageKeys.CCLOUD_AUTH_STATUS,
+      setCCloudAuthStatusStub,
```

Comment:
Updated to stub the call made more closely to this portion of `createSession()`:
https://github.com/confluentinc/vscode/blob/602c95625eefacf5c1288bf32cc61857cbcc54e9/src/authn/ccloudProvider.ts#L188-L192